### PR TITLE
Feature/pcastell/debug viirs paths

### PR DESCRIPTION
**Pull request by patricia-nasa**
-update vx04.py for new logic in VIIRS directory names
-update vx04.py to split up land pixels into DB_DEEP and DB_LAND.  these are two different retrieval pathways

### CODE REVIEW

File: src/pyobs/vx04.py [hunk 1]
```diff
@@ -155,9 +157,10 @@
 translate_sat = {'Suomi-NPP': 'SNPP'}
 
 
-KX = dict ( SNPP_DT_OCEAN = 336,
-            SNPP_DT_LAND  = 335,
-            SNPP_DB_OCEAN  = 334,
+KX = dict ( SNPP_DT_OCEAN = 337,
+            SNPP_DT_LAND  = 336,
+            SNPP_DB_OCEAN  = 335,
+            SNPP_DB_DEEP   = 334,
```
Comment (vbuchard) [inline src/pyobs/vx04.py:163]: Do we need to update the kx values?

**Review by vbuchard — DISMISSED** [review]
Looks good to me. I see the kx values have changed.

File: src/pyobs/vx04.py (hunk 1 above)
Comment (patricia-nasa) [reply src/pyobs/vx04.py:163]: yes, i added a new viirs retrieval called db_deep (before it was only db_land and db_ocean).  so I had to add a new kx value.  i had updated this in AeroApps as well.

Comment (patricia-nasa) [reply src/pyobs/vx04.py]: ok, i have updated is so VSNPPDBO = 334 again, and VSNPPDBD = 335 now.  this should still be consistent with past experiments

Comment (vbuchard) [reply src/pyobs/vx04.py]: Thanks, it will simplify the comparisons :-).

**Pull request merged**